js/manager.js

```diff
@@ -4,13 +4,20 @@ import * as Utils from './utils';
 let loadAlreadyCalled = false;
 let googleGPTScriptLoadPromise = null;
 const registeredSlots = {};
-let pubadsService = null;
 let managerAlreadyInitialized = false;
 const globalTargetingArguments = {};
 
 const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
   setTargetingArguments(data) {
     Object.assign(globalTargetingArguments, data);
+    this.getGoogletag().then((googletag) => {
```

Comment:
tests well on my local, I only found a little issue for  a pattern like the following, which will end up creating a double assignment:
```
...
DFPManager.setTargetingArguments({ foo: 'bar' });
DFPManager.load();
...
```
debugger:
```
Setting targeting attribute foo with value bar for service publisher_ads
Setting targeting attribute foo with value bar for service publisher_ads
```
We can fix this just surrounding this block with the following:
```
if (loadAlreadyCalled === true) {
  this.getGoogletag().then((googletag) => {
    googletag.cmd.push(() => {
      ..
    }
  });
}
```

Thanks!